### DISCUSSION TITLE
Simple machine learning model ideas?

### DISCUSSION THREAD

**Discussion by 735713038455163** [ALIYA2Group/Mod20_Segment_1#6] [Ideas]
https://apizzo1-hindsight-2020.herokuapp.com/
https://docs.google.com/presentation/d/e/2PACX-1vQ9e59BSZeiK2LWKrIjWPhR_mGaKAb4BfJNv[…]8NI8DQ-XHF/pub?start=false&loop=false&delayms=3000&slide=id.p

Here are the links foe examples on final project? Do you have any ideas you would like to share?

**Comment by 735713038455163** [comment]
1) Predictive analytics forecasting future sales of a supermarket or mart basis historical data & events: Although there are statistical forecasting techniques such as ARIMA which can be used to a decent effect; still the inculcation of machine learning in forecasting opens up a lot of other avenues which the traditional statistics technique invariable ignore or assume. We can access pre-defined sales datasets wherein historical sales data for different outlets are given along with other predictor variables so as to design a regression model out of it.
2) Sentiment analysis on social media content: This is also a frequently ventured project specifically by business organizations to understand what the world is saying about their brands, products & services. For us, they can extract social media data from Twitter, Instagram, Facebook etc., and start with an AI-based classification problem which could simply classify a particular content (be it a comment or a tweet) as positive or negative content in terms of the sentiment it is carrying.
3) Artificial-based recommendation engine: Not many users acknowledge or are aware of the fact that the custom recommendation that they get on e-commerce portals, or content-based portals are built on sophisticated and extensive algorithms. For this project, We need to understand the concept of ‘collaborative filtering’ and create correct pairings between user-user, item-item, and user-item so that the recommendations are relevant to the users.
4) We who are planning to carve a rewarding career in the field of financial econometrics can be take up stock price forecast/prediction as a valuable use-case wherein machine learning can be leveraged to a considerable limit. Although the proposition may seem interesting, there are challenges in the successful implementation of a stock price predictor. The plethora of dependencies on which a stock price depends (such as company performance, consumer sentiments, macroeconomic variables, volatility indicators etc.) makes it difficult for analysts to strive for accuracy.
5) Classification or segmentation problems can be solved using machine learning algorithms. In order to implement this, we can access the ‘Iris database’ which is readily available on the web. The data has multiple attributes of flowers of different specifies; and the analyst need to apply appropriate machine learning techniques to classify them in one amongst the three specified species.
6) Dynamic pricing in retail is another hot research topic in the field of machine learning. In this, the price of the product is set depending on the demand of the product (at that point in time) vis-à-vis the supply dynamics.

**Comment by 735713038455163** [comment]
First Segment: Sketch It Out: Decide on your overall project, select your question, and build a simple model. You'll connect the model to a fabricated database, using comma-separated values (CSV) or JavaScript Object Notation (JSON) files, to prototype your idea.

# Presentation 

* Selected topic:

* Reason we selected the topic:

* Description of the source of data:

Minimum of 1,000 data points,using either Kaggle, data.world, government sites, Web scraping. 

* Questions we hope to answer with the data:

**Reply by 735713038455163** [reply]
@amberwnaushahi @maryamt95

**Reply by 735713038455163** [reply]
**Selected topic:** Predicted when polar ice will be gone in the Artic and Antarctic's

**Reason we selected the topic:** Sea ice, which is comprised of frozen ocean water, is found only in the Arctic and Antarctic. The amount of sea ice at each pole changes throughout the year, growing in the winter months and shrinking during the summer months. This visualization on Google Earth shows the minimum sea ice measured for each year from 1979 to 2018. And could expand this to 2022. For the Antarctic, this measurement is taken in March after summer in the Southern Hemisphere. Sea ice has an impact on the global climate from helping to regulate global temperature to affecting ocean currents and providing a habitat for wildlife. We want to explore the catastrophes and aftermath that may occur after that date, and provide useful information to help navagate hard times in the future if we don't do something now. 

**Description of the source of data:**
1. National Snow and Ice Data Center. NSIDC data on Google Earth. Digital media. http://nsidc.org/data/google_earth. Accessed March 1, 2019.
2. Find more data
Minimum of 1,000 data points,using either Kaggle, data.world, government sites, Web scraping.

**Questions we hope to answer with the data:**
- Predicting Instinction of Animals due to Climate Change
- Predicting Animal Extinction
- Fire, Ice & Water
- Fire Zones
- Flood Zones
- Ice Caps

**Comment by maryamt95** [comment]
1.  Relationship between education and income . 
    * hoping to answer :Do students who are economically disadvantaged perform at a lower level academically compared to their non-economically disadvantaged classmates?
2. Risk analysis for investment portfolio
    * hoping to answer - Am I making the  correct investment?
3.  Child service crisis call analysis 
4. Energy consumption and generation over time  
5. Smarter rental prices

**Reply by 735713038455163** [reply]
1. Data sources for 1
https://search.open.canada.ca/en/od/?sort=score%20desc&page=1&search_text=education%20&od-search-keywords=education|education%20training%20and%20learning 
https://data.world/search?context=community&q=education&type=all 
2. might be to large of scope
3. https://search.open.canada.ca/en/od/?sort=last_modified_tdt%20desc&page=1&search_text=child%20crisis
4. https://data.world/datasets/energy
5. https://open.canada.ca/data/en/dataset/499b30cc-15ed-46a9-953e-17187f85be4b

**Reply by amberwnaushahi** [reply]
4. https://cs109-energy.github.io/

**Reply by amberwnaushahi** [reply]
4. https://towardsdatascience.com/forecasting-energy-consumption-using-neural-networks-xgboost-2032b6e6f7e2

**Comment by amberwnaushahi** [comment]
Black Friday Sales Prediction
Want to work on a regression model and expand your feature engineering skills? you can use retail sales data to make predictions about Black Friday sales.  The dataset should contain demographic information for customers (including age, gender, marital status, location, and more) as well as product details and total purchase amounts.

**Reply by amberwnaushahi** [reply]
Weather patterns and predictions to help farm prepare for disaster/incoming storms - based on Yasir's experience in insurance industry, he feels this is very high in demand.

**Comment by 735713038455163** [comment]
- Predicted when polar will be gone in Artic and Antartics https://earth.google.com/web/search/polar+sea/@-83.85598755,44.06260612,-8229a,19936099d,35y,354h,0t,0r/data=CmESNxIgYzQzNjdmOGM3MGY3MTFlOWIyMmViOTAzOTEzZjY4NDIiE3BvbGFyX2ljZV9hbnRhcmN0aWMiJgokCQAAAAAAAAAAEQAAAAAAAAAAGQAAAAAAAAAAIQAAAAAAAAAA
- NSIDC Data on Google Earth | National Snow and Ice Data Center
https://nsidc.org/data/google_earth
-https://nsidc.org/cryosphere/seaice/environment/trends.html
-https://nsidc.org/data/google_earth

**Comment by amberwnaushahi** [comment]
Climate Change:

- Predicting Animal Extinction 

- Fire, Ice & Water
    - Fire Zones
    - Flood Zones
    - Ice Caps

**Reply by maryamt95** [reply]
I have written a query from the  database at this location :
https://www.worldwildlife.org/publications/wildfinder-database

to extract list of species to be extinct by region . this is the data . Targeting Antarctic and Arctic regions , we can assume that is because of Global warming . 
[Endangered species by region .xlsx](https://github.com/ALIYA2Group/Mod20_Segment_1/files/7836157/Endangered.species.by.region.xlsx)

**Reply by maryamt95** [reply]
Not sure if ERD is needed , but I had done this using MS Access , since the database was .MDR format 
![image](https://user-images.githubusercontent.com/89698299/148705404-65ae92a2-bddd-4172-a863-a7167f90d288.png)

**Comment by maryamt95** [comment]
looking at the data from this link :  https://nsidc.org/data/masie/browse_regions
attached  is what I have found . checking the FAQ - the data is available for Northern Hemisphere only 
![image](https://user-images.githubusercontent.com/89698299/148706553-086cf40d-10c3-4f23-87b9-f1e660df8dfa.png)
[masie_4km_allyears_extent_sqkm.csv](https://github.com/ALIYA2Group/Mod20_Segment_1/files/7836241/masie_4km_allyears_extent_sqkm.csv)

**Comment by maryamt95** [comment]
data for the ice index .Not sure how we can use this data to draw a path on a map . 
Like this one 

https://nsidc.org/data/seaice_index/

[N_seaice_extent_daily_v3.0.csv](https://github.com/ALIYA2Group/Mod20_Segment_1/files/7836523/N_seaice_extent_daily_v3.0.csv)
[S_seaice_extent_daily_v3.0.csv](https://github.com/ALIYA2Group/Mod20_Segment_1/files/7836524/S_seaice_extent_daily_v3.0.csv)

**Comment by maryamt95** [comment]
I had merged the two files above and re-factored some codes . this is what I got 
[seaice_extent_daily_v3.0.csv](https://github.com/ALIYA2Group/Mod20_Segment_1/files/7841880/seaice_extent_daily_v3.0.csv).

It is not allowing me to update the code , I will add it in github .
